packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -160,6 +160,15 @@ export const prodParasPolkadot: EndpointOption[] = [
       Dwellir: 'wss://hydradx-rpc.dwellir.com'
     }
   },
+  {
+    info: 'integritee',
+    homepage: 'https://integritee.network',
+    paraId: 2039,
+    text: 'Integritee Shell on Polkadot',
```

Comment:
Not convinced by "on Polkadot" (it displays on the Polkadot section, so it is not confusing)

Reply:
OK, removed "on Polkadot"